### DISCUSSION TITLE
Europe based package repositories / mirrors

### DISCUSSION THREAD

**Discussion by belmont** [linuxdeepin/developer-center#10423] [Features Request & Ideas | 特性请求 & 头脑风暴]
You put efforts for European language translations but there is no package repository mirror in Europe. Many countries are blocking traffic between China and Europe and currently with repository in China your OS is unusable. It fails to connect. Please allow Europe based mirrors and publish here who we can contact for offering hosting.

**Comment by BLumia** [comment]
> Please allow Europe based mirrors and publish here who we can contact for offering hosting.

Usually you don't really need a permission to mirror deepin's package repo.

See also: https://www.deepin.org/zh/mirrors/setup-mirror-sites/ (P.s. There should be an English-language version of that page, but currently the English language page is blank. Will reach out related team to check if there something went wrong).

If you (or anyone) are interested in setting up a mirror, feel free to reach us out by creating a discussion thread (like this one), an issue or contact us via email (you can find the address on our official website) :)

> .. and publish here who we can contact for offering hosting.

There are a lot of existing mirrors, but I'm not sure if these mirrors are still working. See: https://www.deepin.org/en/mirrors/packages/

edit: see also: https://www.deepin.org/en/mirrors/
